src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -536,9 +551,14 @@ public void testEnrollLinesChecking() throws Exception {
 
         enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInputWithComment;
 
-        invalidInfo = getInvalidityInfoInEnrollLines(enrollLines, courseId);
+        invalidInfoString = null;
+        try {
+            studentsLogic.createAndValidateStudents(enrollLines, courseId);
+        } catch (EnrollException e) {
+            invalidInfoString = e.getMessage();
+        }
 
-        assertEquals(0, invalidInfo.size());
+        assertNull(invalidInfoString);
```

Comment:
This whole block 554-561 can be simplified to:

```java

studentsLogic.createAndValidateStudents(enrollLines, courseId);

```

If the method call throws an exception, the test will fail at this point, which is what we want anyway.